bin/git/git-node-apply

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+'use strict';
+
+const PRID = parseInt(process.argv[2]);
+const Request = require('./../../lib/request');
+const paths = require('./../../lib/paths');
```

Comment:
I think for both `Request` and `paths` you can remove `./`
nit: `../../lib/request` and `../../lib/paths`